bridge/evm/contracts/utils/BridgeCommon.sol

```diff
@@ -3,22 +3,29 @@ pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "./interfaces/IBridgeTokens.sol";
+import "../interfaces/IBridgeCommon.sol";
 
-/// @title BridgeTokens
-/// @notice This contract manages the supported tokens of the SuiBridge. It enables the contract owner
-/// (intended to be the SuiBridge contract) to add and remove supported tokens. It also provides functions
-/// to convert token amounts to Sui decimal adjusted amounts and vice versa.
-contract BridgeTokens is Ownable, IBridgeTokens {
```

Comment:
why was a Ownable before and no longer needed now?

Reply:
It was owneable when I had the logic to update the "supportedTokens" mapping but now that we're planning on deploying a new contract when we're ready to support adding new tokens, it's not needed. I'm wondering if we want to just make this contract committee upgradeable 🤔

Reply:
that sounds like a reasonable idea.
is solidity interface upgradeable?

Reply:
Not directly, but we'll introduce new interface versions when we upgrade the referenced contract. Any contracts that need to reference the upgraded contracts functionality will also need to be upgraded to reference the new contract version interface.